web/src/test/java/se/inera/intyg/rehabstod/service/sjukfall/GetSickLeaveSummaryServiceTest.java

```diff
@@ -75,14 +75,14 @@ public class GetSickLeaveSummaryServiceTest {
     @BeforeEach
     void setup() {
         doReturn(new GetActiveSickLeavesResponseDTO(sickLeaves, false, false))
-            .when(getActiveSickLeavesResponseService).get(any(SickLeavesFilterRequestDTO.class), any(boolean.class));
+            .when(getActiveSickLeavesResponseService).get(any(SickLeavesFilterRequestDTO.class), any(boolean.class), false);
```

Comment:
This comment is not specific to this line... 

Include a test that verifies explicitly that NO PDL logging should be sent.